### ISSUE TITLE
Designer should show tables from other databases by default

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#13587]
When using the new feature #5827 to allow Designer to show tables from other databases, I can't save a layout that shows the other tables. Further, I think by default Designer should show those tables if a relationship exists to a table in another database.

Perhaps we should alter the color of the tables shown to indicate they're from another database.

Possibly related to #13370 and #12900.

**Comment by anantjain6** [comment]
Hi, Did this issue is still open to work ?

**Comment by ibennetch** [comment]
@anantjain6 Yes, it doesn't look like anyone else is working on it right now.

**Comment by Bournvita1998** [comment]
I am able to add the tables from other databases but I am not able to form the relations between the tables(one of the current database and other of the other database). It shows errors like "error: relationship already exists", "FOREIGN KEY relationship could not be added", etc. Basically, I am not able to form the relationship.
Would be great if someone could share the steps to reproduce this from scratch.
